static/skin/taskbar.js

```diff
@@ -3,6 +3,39 @@ function htmlDecode(input) {
     return doc.documentElement.textContent;
 }
 
+function handleScrollFor520Width() {
+  let lastScrollTop = 0;
+  let delta = 5;
+  let didScroll = false;
+
+  window.onscroll = function (e) {
+    didScroll = true;
+  }
+
+  setInterval(function() {
+    if (didScroll) {
+      hasScrolled();
+      didScroll = false;
+    }
+  }, 250);
+
+  function hasScrolled() {
+    let st = document.documentElement.scrollTop || document.body.scrollTop;
+    const kiwixToolBar = document.querySelector('#kiwixtoolbar');
```

Comment:
Move `kiwixToolBar` into the outer function

Reply:
:white_check_mark: